Integrations/BeyondTrust_Password_Safe/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
+Fixed an issue where stored credentials were using a non-unique identifier.
```

Comment:
lets mention here that this change the previous behavior - work with @kirbles19 on that

Reply:
what do you mean, are you referring to the fact that this will break backward compatibility? You want me to specifically mention that in the changelog?

Reply:
yea